src/ProcessMaker/Nayra/Bpmn/ConditionedTransition.php

```diff
@@ -24,12 +24,12 @@ class ConditionedTransition implements TransitionInterface, ConditionedTransitio
     /**
      * Condition required to transit the element.
      *
-     * @param TokenInterface $token
-     * @param ExecutionInstanceInterface $executionInstance
+     * @param TokenInterface|null $token
+     * @param ExecutionInstanceInterface|null $executionInstance
      *
      * @return mixed
      */
-    public function assertCondition(TokenInterface $token = null, ExecutionInstanceInterface $executionInstance)
+    public function assertCondition(TokenInterface $token = null, ExecutionInstanceInterface $executionInstance = null)
     {
         $condition = $this->condition;
         return $condition($executionInstance->getDataStore()->getData());
```

Comment:
as executionInstance can be null, this line can throw an error

Reply:
Fixed, when the $executionInstance is null the $dataSource should be the global one:

```
        $dataStore = $executionInstance ? $executionInstance->getDataStore()
            : $this->getOwnerProcess()->getEngine()->getDataStore();
```